charts/kube-otel-stack/values.yaml

```diff
@@ -163,6 +163,12 @@ metricsCollector:
         - key: collector.name
           value: "${KUBE_POD_NAME}"
           action: insert
+        - key: job
+          from_attribute: service.name
+          action: insert  
+        - key: k8s.pod.ip
+          from_attribute: "net.host.name"
```

Comment:
Hmm I'm nervous about this addition because I feel like I've seen instances where `net.host.name` is not an ip address. i.e. its a string which I assume gets resolved by kube into an ip address- is that still okay? (e.g. here is a kube metric that has a string for `net.host.name` https://app.lightstep.com/dev-mohosman/notebooks/net-host-name-not-always-an-ip-address/FzBq9J4Z)

Reply:
ah i can remove this one, really i just want the job one back. i think it's fine if net.host.name isn't an ip address, i think it can resolve to a hostname set in /etc/hosts but regardless, i'm removing this :)

Reply:
I'm fine either way just wanted us to be aware that this is the case!